ceph_deploy/util/pkg_managers.py

```diff
@@ -206,6 +210,18 @@ def clean(self):
         """Clean metadata/cache"""
         raise NotImplementedError()
 
+    def add_repo_gpg_key(self, url):
+        """Add given GPG key for repo verification"""
+        raise NotImplementedError()
+
+    def add_repo(self, name, url, gpg_url):
```

Comment:
Are you saying to handle the existing positional args args inside `**kw` instead?

Adding `**kw` seems like a good idea.  My personal preference is to make *required* args be clear/obvious by making them positional.  But it is just that -- a personal preference.  So I would lobby for adding `**kw` and leaving the existing ones.  What say you?  :)